<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaticSectorsDialog</class>
 <widget class="QDialog" name="StaticSectorsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select sectors</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Additionally shown sectors</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="btnSelectNone">
          <property name="text">
           <string>Select &amp;none</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnSelectAll">
          <property name="text">
           <string>Select &amp;all</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listWidgetSectors">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="movement">
         <enum>QListView::Static</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>StaticSectorsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
